index.html

```diff
@@ -4677,98 +4722,165 @@ <h2>Internationalization Considerations</h2>
+JSON, JSON-LD, or similar data formats to ensure the broadest interoperability
+across syntaxes. The design pattern is a result of extensive collaboration
+between the W3C Internationalization Activity, the Linked Data Community, and
+developers that primarily utilize JSON and CBOR to publish information. The
+pattern is useful for the vast majority of language and base direction use
+cases that implementers of this specification will need to support.
       </p>
 
-      <ul>
-        <li>
-[[!JSON-LD]] supports the expression of language information using the
-<code>@language</code> feature, as described in [[!JSON-LD]] 1.1,
-<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">Section 4.2.3: String Internationalization</a>.
-        </li>
-        <li>
-[[!JSON-LD]] also supports the expression of language information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
-<a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
-This mechanism enables developers to use the internationalization features
-present in HTML to express language information. For example, consider the
-following key-value pair:
+      <p>
+The general design pattern is to use the following markup template when
+expressing a text string that is tagged with a language and, optionally, a
+specific text direction:
+      </p>
 
-          <pre>
-"nameHtml": "&lt;span lang="jp">和美&lt;/span>"
-          </pre>
+      <pre class="example nohighlight" title="Design pattern for natural language strings">
+"<a>property</a>": {
+  "value": "<span class="highlight">The string value</span>",
+  "language: "<code>LANGUAGE</code>-x-dir-<code>DIRECTION</code>"
+}
+      </pre>
 
-          <p>
-If the JSON-LD Context expressed that any value associated with the
-<code>nameHtml</code> property as being of type <code>rdf:HTML</code>, then
-software agents rendering the property can deterministically identify the
-language in use.
-          </p>
-        </li>
-      </ul>
-    </section>
+      <p>
+An example utilizing the design pattern above is shown below to express the
+human-readable text "revoked" in the English language without specifying a
+text direction:
+      </p>
 
-    <section class="informative">
-      <h3>Text direction</h3>
+      <pre class="example nohighlight" title="Expressing natural language text as English">
+"statusReason": {
+  "value": "<span class="highlight">Revoked</span>",
+  "language: "<code>en</code>"
+}
+      </pre>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-to be able to explicitly encode the text direction. JSON and its derived formats
-use the UTF-8 encoding of [[Unicode]] for serializing and transmitting text.
-This enables text direction to be directly determined in simple cases,
-but the mechanism is insufficient for general use.
+The next example expresses the human-readable text "revoked" in the Arabic
```

Comment:
`human-readable text "revoked"`
-> `human-readable English text "revoked"`

Reply:
`human-readable text "revoked"`
-> `English text "revoked"`

Reply:
fine by me

Reply:
Fixed in e544fe83ea9600947ae5f2a45925878abd6e8972.